### ISSUE TITLE
libmraa.so not found

### ISSUE THREAD

**Issue by darley02** [radxa/Android-MraaDemo#1]
Hello,

When I clone and try running this project in android studio, I get the following error:

W/System.err: Native code library failed to load.
    java.lang.UnsatisfiedLinkError: dlopen failed: library "libmraa.so" not found

Is there a separate libmraa.so lib that should be downloaded for use with the included libmraajava.so in mraa-2.1.0.arr?

Thanks,
Akin

**Comment by antjpar** [comment]
I have the same issue, running on the Rockchip Android 10. I see that the libmraa.so is in the /system/lib64 partition but is not being loaded. 

Not sure if something needs to be done on the Android OS or in the Application to correctly use the MRAA functions

**Comment by cx-sawyer** [comment]
I'd like to invite you to try Android11 for Rock PI 4.
https://github.com/radxa/manifests/releases/tag/Rock-android11-20211115_1851
Meanwhile please use "rockchip_android11" branch of Android-MraaDemo.

**Comment by cx-sawyer** [comment]
> I have the same issue, running on the Rockchip Android 10. I see that the libmraa.so is in the /system/lib64 partition but is not being loaded.
> 
> Not sure if something needs to be done on the Android OS or in the Application to correctly use the MRAA functions

You are right, please try to edit /system/etc/public.libraries.txt, add libmraa.so to the end of file in a new line.
Steps:
1. adb root
2. adb disable-verity
3. adb remount
4. adb shell 
5. busybox vi /system/etc/public.libraries.txt
6. add "libmraa.so" in a new line
7. save and reboot
